JSDemos/Demos/TreeList/FilterPanel/description.md

```diff
@@ -1 +1,7 @@
-This example shows how to&nbsp;filter data using the filter panel. You can use its check box to&nbsp;enable/disable the current filter expression, and clicking on&nbsp;this expression opens the integrated filter builder. Note that changes made in&nbsp;it&nbsp;are reflected in&nbsp;the filter row and header filter, and vice versa.
\ No newline at end of file
+The [filterPanel](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterPanel/) is a UI element that displays the combined filter in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterValue) property and consists of filters that users apply in the UI elements ([filter row](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterBuilder)). Users can click the checkbox in the filter panel to apply or deactivate the current filter.
```

Comment:
I wouldn't use "UI element" the first time, so that it doesn't sounds that much as a "definition". The second time it's good, because we need a word that references all other ways to set a filter. All in all - a couple of minor changes.

Also I'm removing repetition of the word "filter" - you can use "conditions", "criteria"... 

```suggestion
The [filterPanel](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterPanel/) displays the combined filter in the bottom of the TreeList. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterValue) property and consists of conditions that a user applied in the following UI elements: [filter row](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#filterBuilder). Users can deselect the checkbox in the filter panel to temporarily deactivate the current filter.
```